### PR TITLE
add edit link for user own profile and onboarding preview page

### PR DESCRIPTION
**Pull request by dianaguimaraes**
Tickets:
- https://asaphub.atlassian.net/browse/GP2-146
- https://asaphub.atlassian.net/browse/GP2-158

### CODE REVIEW

File: apps/gp2-frontend/src/hooks/__tests__/useSelectAvatar.test.tsx [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+import { Auth0Provider, WhenReady } from '../../auth/test-utils';
+import { getUser, postUserAvatar } from '../../users/api';
+import { refreshUserState } from '../../users/state';
+import { useSelectAvatar } from '../useSelectAvatar';
+
+jest.mock('../../users/api');
+jest.mock('browser-image-compression');
+
+const mockGetUser = getUser as jest.MockedFunction<typeof getUser>;
+// const mockPatchUser = patchUser as jest.MockedFunction<typeof patchUser>;
```
Comment (jhonny-m) [inline apps/gp2-frontend/src/hooks/__tests__/useSelectAvatar.test.tsx:24]: we can remove this line

Comment (dianaguimaraes) [reply apps/gp2-frontend/src/hooks/__tests__/useSelectAvatar.test.tsx:24]: yeah, definitely! 😄

File: apps/gp2-frontend/src/onboarding/Preview.tsx [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+        <Route path={editBiography({}).$}>
+          <BiographyModal {...userData} {...commonModalProps} />
+        </Route>
+        <Route path={editKeywords({}).$}>
+          <KeywordsModal {...userData} {...commonModalProps} />
+        </Route>
+        <Route path={editQuestions({}).$}>
+          <OpenQuestionsModal
+            {...userData}
+            backHref={onboarding({}).additionalDetails({}).$}
```
Comment (jhonny-m) [inline apps/gp2-frontend/src/onboarding/Preview.tsx:95]: update to the `commonModalProps`

Comment (dianaguimaraes) [reply apps/gp2-frontend/src/onboarding/Preview.tsx:95]: 👍🏻

**Review by jhonny-m — APPROVED** [review]
looks good 👍

**Pull request merged**